components/Animation.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import LottieView from "lottie-react-native"
+
+
+const Animation = () => {
+  return (
+    <LottieView style={{ width: 350, height: 350, backgroundColor: "#F8EFBA" }} source={require('../assets/walking.json')} autoPlay loop />
+  )
+}
```

Comment:
nice that you used a Lottie animation! Makes such a big difference having a nice animation like that! ⭐